src/main/java/com/chaos/invoicify/dto/InvoiceDto.java

```diff
@@ -0,0 +1,19 @@
+package com.chaos.invoicify.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class InvoiceDto {
+    String invoiceName;
+    String companyName;
+    String invoiceDate;
```

Comment:
Replace invoiceName with invoiceNumber. Add totalCost and modifiedDate.
Replace invoiceDate with createDate. createDate and modifiedDate should be of Date type

Reply:
Done